modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -434,62 +349,56 @@ wrapper.java.additional.<n>=-Danypoint.platform.analytics_multiple_consumers_ena
 [WARNING]
 If you modify your `wrapper.conf` file as described above but don't assign an external destination for your data (as you can do via the Runtime Manager UI), then this analytics data will be stored in a queue in the server where the API gateway is being run and could pile up to the point of crashing the system.
 
-[discrete.view]
-== Integrating API Analytics with Splunk and ELK
 
-Once you've configured your API gateway, you can now return to Runtime Manager and see that your servers have some additional options in their menu.
+=== Integrate API Analytics with Splunk and ELK
+
+After you've configured your API gateway, you can now return to Runtime Manager and see that your servers have some additional options in their menu.
 
 image::sending-data-from-arm-to-external-monitoring-software-api-analytics.png[]
 
 You can now set up the sending of API analytics to both Splunk and ELK, you configure exactly in the same way as you do when sending business events to them. See <<Integrating with an ELK Stack, Integrating with an ELK Stack>> and <<Integrating with Splunk, Integrating with Splunk>>.
 
-[discrete.view]
-== Payload Formats
 
-If you intend to export your payload's contents to be logged, keep in mind that not all formats can be exported. See xref:about-logging-of-payload-formats.adoc[About Logging of Payload Formats] for a full reference.
+This section can be removed, I think VVVV
```

Comment:
should this be an internal note? seems like a committed change.

Reply:
Looks like you've made a comment on an earlier checkin. Subsequently removed.